internal/cephfs/util.go

```diff
@@ -32,38 +30,13 @@ import (
 
 type volumeID string
 
-func execCommand(ctx context.Context, program string, args ...string) (stdout, stderr []byte, err error) {
-	var (
-		cmd           = exec.Command(program, args...) // #nosec:G204, not called with user specified parameters.
-		sanitizedArgs = util.StripSecretInArgs(args)
-		stdoutBuf     bytes.Buffer
-		stderrBuf     bytes.Buffer
-	)
-
-	cmd.Stdout = &stdoutBuf
-	cmd.Stderr = &stderrBuf
-
-	util.DebugLog(ctx, "cephfs: EXEC %s %s", program, sanitizedArgs)
-
-	if err := cmd.Run(); err != nil {
-		if cmd.Process == nil {
-			return nil, nil, fmt.Errorf("cannot get process pid while running %s %v: %w: %s",
-				program, sanitizedArgs, err, stderrBuf.Bytes())
-		}
-		return nil, nil, fmt.Errorf("an error occurred while running (%d) %s %v: %w: %s",
-			cmd.Process.Pid, program, sanitizedArgs, err, stderrBuf.Bytes())
-	}
-
-	return stdoutBuf.Bytes(), stderrBuf.Bytes(), nil
-}
-
 func execCommandErr(ctx context.Context, program string, args ...string) error {
-	_, _, err := execCommand(ctx, program, args...)
+	_, _, err := util.ExecCommand(program, args...)
```

Comment:
Make ctx a parameter of execcommand it helps for debugging, as we log request if and failure

Reply:
I think logging should be part of the function that tries to execute the command. It should only be needed to log the command in case an error occurs, if it is not expected/handled.

Would you be fine with additional logging in cases where it is not done sufficiently?

Reply:
When debugging in larger system logging with request id always helps. 

Yeah we also need to log the commands what we are executing

Reply:
Ctx is unused in this function

Reply:
So, do you want me to drop context from all command execution calls, and add logging in the function that calls `util.ExecCommand()`, or do you prefer logging in `util.ExecCommand()` itself, regardless of the success/failure of the command?

Reply:
I prefer to log the command with ctx regardless of success/failure in ExecCommand function

Reply:
This should be done now, passing the context.Context when suitable (only 2 occasions where it isn't needed).